userspace/libscap/scap.c

```diff
@@ -1464,6 +1465,57 @@ uint64_t scap_get_driver_schema_version(scap_t* handle)
 	return 0;
 }
 
+void scap_gethostname(scap_t* handle)
+{
+	gethostname(handle->m_machine_info.hostname, sizeof(handle->m_machine_info.hostname) / sizeof(handle->m_machine_info.hostname[0]));
+	char *env_hostname;
+	env_hostname = getenv("FALCO_HOSTNAME");
```

Comment:
The env var must not be `FALCO_` prefixed, but we'll need to make that a CMake overridable variable. Of course, for Falco as a consumer the name could be `FALCO_HOSTNAME`, but other libs consumers might want to have different namings.

Example of how we do this already for another env var:
- https://github.com/falcosecurity/libs/blob/b9591d68f44924f49bc9248638377b44a5328f56/userspace/libscap/scap.c#L1316
- https://github.com/falcosecurity/libs/blob/b9591d68f44924f49bc9248638377b44a5328f56/userspace/libscap/CMakeLists.txt#L44